payments/views.py

```diff
@@ -87,16 +86,20 @@ def stripe_webhook(request, customer_email=None):
     if event.type == 'checkout.session.completed':
         completed_checkout_session = event.data.object
         amount_received = int(completed_checkout_session.amount_total / 100)
-        user_profile_id = completed_checkout_session.metadata.user_profile_id
-        profile = Profile.objects.get(pk=user_profile_id)
-        profile.money += amount_received
-        profile.save()
+        user_profile_pk = completed_checkout_session.metadata.user_profile_pk
+        try:
+            profile = Profile.objects.get(pk=user_profile_pk)
+            profile.money += amount_received
+            profile.save()
+        except profile.DoesNotExist:
+            print("An exception occurred")  
```

Comment:
to jest problemowa sytuacja - komus zabralo kase z konta, ale nie dostal doladowania w aplikacji.
Wydaje mi sie ze moznaby napisac templatke typu wystapil blad, skontaktuj sie z administratorem czy cos takiego (tresc pewnie do dogrania z Damianem) i zrobic redirect w tym miejscu, zeby bylo bardziej akuratnie.

Reply:
ew. mozna po prostu zrobic z tego nowgo taska w clickupie i ktos to zgarnie